tools/staticbuild/build_wheel.sh

```diff
@@ -18,11 +18,30 @@
 # under the License.
 
 # This script builds the wheel for binary distribution and performs sanity check.
+# To be used only on vanilla Ubuntu-14.04 to ensure backwards compatibility
+# of all dependencies of MXNet.
+
+set -e
+
+echo $(pwd) > pwd.txt
+export SRC="`sed -E 's/(.*incubator\-mxnet)(.*)/\1/g' pwd.txt`"
```

Comment:
What is this obscure thing doing?

Reply:
it sets the SRC to location to incubator-mxnet root folder.

Reply:
Maybe this is better? Also why write a file and polute the disk?

https://github.com/apache/incubator-mxnet/blob/master/ci/util.py#L22